aepp/flowservice.py

```diff
@@ -1296,6 +1298,22 @@ def getLandingZoneContainerName(
         """
         return self.getLandingZoneContainer(dlz_type=dlz_type)["containerName"]
 
+    def getLandingZoneStorageName(
+            self,
+            dlz_type: str = "user_drop_zone"
+    ) -> str:
+        """
+        Returns the name of the DLZ storage corresponding to this type.
+        Arguments:
+            dlz_type : OPTIONAL : The type of DLZ container - default to "user_drop_zone" but can be "dlz_destination"
+        """
+        response = self.getLandingZoneContainer(dlz_type=dlz_type)
+        if self._isAzureLandingZone(response):
+            return response["containerName"]
+        else:
+            return response["dlzPath"]
```

Comment:
I think you meant to return `response["dlzPath"]["dlzFolder"]` right? Because this is just going to return a dictionary based on the existing spec.